website/docs/r/snat.html.markdown

```diff
@@ -28,5 +32,7 @@ The following arguments are supported:
 * `edge_gateway` - (Required) The name of the edge gateway on which to apply the SNAT
 * `external_ip` - (Required) One of the external IPs available on your Edge Gateway
 * `internal_ip` - (Required) The IP or IP Range of the VM(s) to map from
+* `network_type` - (Optional; *v2.4+*) Type of the network on which to apply the NAT rule. Possible values org or ext. *`network_type` will be a required field in the next major version.*
```

Comment:
Can we briefly mention the default here.

Reply:
no default here